aiidalab/utils.py

```diff
@@ -19,7 +19,7 @@
 from .config import AIIDALAB_REGISTRY
 
 logger = logging.getLogger(__name__)
-
+_TTL_CACHE = TTLCache(maxsize=32, ttl=60)
```

Comment:
Since this cache is used for a specific purpose and becomes part of the module's API, I'd suggest to name this to something meaningful:
```suggestion
FIND_INSTALLED_PACKAGES_CACHE = TTLCache(maxsize=32, ttl=60)
```